drivers/hmis_csv_twenty_twenty_two/app/models/hmis_csv_twenty_twenty_two/loader/loader_concern.rb

```diff
@@ -12,7 +14,7 @@ module LoaderConcern
       belongs_to :loader_log, optional: true
 
       def hmis_data
-        @hmis_data ||= slice(*self.class.hmis_structure.keys)
+        @hmis_data ||= slice(*self.class.hmis_structure.keys & attributes.keys)
```

Comment:
seeing some crashes here due to values in `hmis_structure.keys` that were not in the model's attrs